stable/spinnaker/values.yaml

```diff
@@ -75,6 +75,10 @@ halyard:
   # env:
   #   - name: DEFAULT_JVM_OPTS
   #     value: -Dhttp.proxyHost=proxy.example.com
+  customCerts:
+    ## Enable to override the default cacerts with your own one
+    enabled: false
+    secretName: my-cacerts
```

Comment:
could you rename this to something more meaningful, e.g. `custom-cacerts`

Reply:
Done